### DISCUSSION TITLE
Issue: When using Htmlable for the global search html is not applying for recent searches

### DISCUSSION THREAD

**Discussion by melvinmihaylovnulabg** [CharrafiMed/global-search-modal#65] [Q&A] [answered]
When html is used for the getGlobalSearchResultTitle, the html is not applying for the recent searches.

**Comment by CharrafiMed** [comment, marked as answer]
unfortunately the  plugin  doesn't support persisting Htmlable parts in the recent search, 
however, could you share your scenario maybe I will add support for it later

**Reply by melvinmihaylovnulabg** [reply]
![image](https://github.com/user-attachments/assets/56d82e72-73e3-4274-b7ab-d491ef2c6bdf)
![image](https://github.com/user-attachments/assets/89562cbe-46da-4b62-9797-f453094d751f)
Yes please, this will be amazing! In the getGlobalSearchResultTitle in any resource you can pass html tags, and when searching in the modal everything is okay, they appear with the html applied, but only for recent the html tags are visible and no html is shown, i am sure it's an easy fix so thank you for the consideration!

**Comment by CharrafiMed** [comment]
HEY @melvinmihaylovnulabg !
install this version https://github.com/CharrafiMed/global-search-modal/releases/tag/v3.4.2

**Reply by melvinmihaylovnulabg** [reply]
@CharrafiMed Thank you for the fast response! It is perfect now!
